docs/vultr.md

```diff
@@ -0,0 +1,31 @@
+# Variomedia
```

Comment:
The typical mistake I do 50% of the time 😄 

```suggestion
# Vultr
```

Reply:
Fixing